src/container/srv_target.c

```diff
@@ -1706,7 +1713,7 @@ cont_close_hdl(uuid_t cont_hdl_uuid)
 		D_ASSERT(cont_child->sc_open > 0);
 		cont_child->sc_open--;
 		if (cont_child->sc_open == 0) {
-			dtx_batched_commit_deregister(cont_child);
+			dtx_cont_close(cont_child);
 			cont_stop_dtx_reindex_ult(cont_child);
```

Comment:
Same question, can we move the cont_stop_dtx_reindex_ult() into dtx_cont_close()?